scripts/variantstore/scripts/hail_gvs_import.py

```diff
@@ -4,14 +4,10 @@
 
```

Comment:
all the changes in this file are refactoring to pull stuff out to `hail_gvs_util.py`.